### PR TITLE
Ldap plugin ported

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpldap/src/main/java/com/agiletec/plugins/jpldap/apsadmin/global-messages_en.properties [hunk 1]
```diff
@@ -36,4 +36,5 @@ jpldap.hookpoint.configSystemParams.userPasswordAlgorithm.tooltip=Default value
 
 jpldap.label.userPasswordAlgorithm.plain=Plain Text
 jpldap.label.userPasswordAlgorithm.md5=MD5
-jpldap.label.userPasswordAlgorithm.sha=SHA
\ No newline at end of file
+jpldap.label.userPasswordAlgorithm.sha=SHA
+jpldap.title.help=The LDAP plugin lets Entando to authenticate users from LDAP server and increase the base of system users.
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpldap/src/main/java/com/agiletec/plugins/jpldap/apsadmin/global-messages_en.properties]: please @DMinnai amend the grammar

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

**Pull request merged**